common/lib/xmodule/xmodule/modulestore/tests/test_split_modulestore_bulk_operations.py

```diff
@@ -444,6 +448,17 @@ def test_get_definitions(self, search_ids, active_ids, db_ids):
             else:
                 self.assertNotIn(db_definition(_id), results)
 
+    def test_get_definitions_doesnt_update_db(self):
+        test_ids = [1, 2]
+        db_definition = lambda _id: {'db': 'definition', '_id': _id}
+
+        db_definitions = [db_definition(_id) for _id in test_ids]
+        self.conn.get_definitions.return_value = db_definitions
+        self.bulk._begin_bulk_operation(self.course_key)
+        self.bulk.get_definitions(self.course_key, test_ids)
+        self.bulk._end_bulk_operation(self.course_key)
+        self.assertFalse(self.conn.insert_definition.called)
```

Comment:
**Note to reviewers:** I have verified that this test breaks on master (without the changes in this PR).